src/js/components/Overview/charts/BentoPie.js

```diff
@@ -41,8 +41,9 @@ function BentoPie({ data, height }) {
 
   // Changing immutable data to mutable data
   data = [...data];
-  // remove "missing" field if zero
+
   data = data.filter((e) => !(e.x === 'missing'));
+  data = data.map((d) => ({ ...d, x: t(d.x) }));
```

Comment:
I would avoid these reassignments; imo its not any less readable in the form of:

```js
data = [...data]
    .filter((e) => !(e.x === 'missing'))
    .map((d) => ({ ...d, x: t(d.x) }));
```
with indenting. In general avoiding mutation of the variable (reference) value is good, although this does get mutated later a bunch anyway...